sample-compose-benchmark/src/main/java/com/airbnb/lottie/sample/compose/benchmark/LottieBaselineBenchmark.kt

```diff
@@ -78,25 +78,22 @@ class LottieBaselineBenchmark {
         }
     }
 
-    @OptIn(ExperimentalTime::class)
     private fun UiScrollable.waitUntilReady() {
         this.waitUntil {
             // We know that there are at least 9 children
             childCount >= EXPECTED_ITEM_INDEX_COUNT
         }
     }
 
-    @OptIn(ExperimentalTime::class)
-    private fun UiObject.clickAndWait(maxWaitTime: Duration = Duration.seconds(5)) {
+    private fun UiObject.clickAndWait(maxWaitTime: Duration = 5.seconds) {
```

Comment:
It seems like these parts of the project aren't invoked on the CI that's why the main branch contains non-compiling code